inc/composer/class-docker-compose-generator.php

```diff
@@ -222,7 +222,7 @@ protected function get_service_nginx() : array {
 					'traefik.protocol=https',
 					'traefik.docker.network=proxy',
 					"traefik.frontend.rule=HostRegexp:{$this->hostname},{subdomain:[a-z.-_]+}.{$this->hostname}",
-					'traefik.backend=nginx',
+					"traefik.backend=nginx-{$this->hostname}",
```

Comment:
Just to confirm this still works ok? I'm not sure what character restrictions and things there are for these values and whether that's a concern.

Reply:
It didn't, which is why I pushed another commit to use project_name :slightly_smiling_face: